models/Comment.php

```diff
@@ -1,126 +1,139 @@
 <?php
-// Permet d'avoir accès à la base de données
-require_once "Model.php";
 
 // Cette classe sert à manipuler tout ce qui touche aux commentaires
-class Comment extends Model
+class Comment
 {
-// FONCTIONS POUR LE FRONTEND
-    // Fonction qui récupère les commentaires d'un article
-    function get_comments_blog(){
-        $req = $this->db->query("SELECT * FROM comments WHERE article_id = '{$_GET['id']}' AND seen = 1 ORDER BY date DESC");
-        $results = [];
-        
-        while($rows = $req->fetchObject()){
-            $results[] = $rows;
-        }
-        
-        return $results;
+    private $id;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)